cli/sql.go

```diff
@@ -67,11 +133,21 @@ func runInteractive(db *sql.DB, dbURL string) (exitErr error) {
 	if len(fullPrompt) == 0 {
 		fullPrompt = " "
 	}
-	continuePrompt := strings.Repeat(" ", len(fullPrompt)-1) + "-"
+
+	continuePrompt = strings.Repeat(" ", len(fullPrompt)-1) + "-"
 
 	fullPrompt += "> "
 	continuePrompt += "> "
 
+	return fullPrompt, continuePrompt
+}
+
+// runInteractive runs the SQL client interactively, presenting
+// a prompt to the user for each statement.
+func runInteractive(db *sql.DB, dbURL string) (exitErr error) {
+
```

Comment:
What's with the blank lines at the start of functions? That isn't a style that we follow and it looks disconcerting to see it on some functions but not others.